contracts/AutomationBot.sol

```diff
@@ -128,8 +139,9 @@ contract AutomationBot is BotLike, ReentrancyGuard {
 
         address commandAddress = getCommandAddress(triggerType);
         if (replacedTriggerId != 0) {
-            (bytes32 replacedTriggersHash, address originalCommandAddress, ) = automationBotStorage
-                .activeTriggers(replacedTriggerId);
+            TriggerRecord memory rec = activeTriggers[replacedTriggerId];
+            bytes32 replacedTriggersHash = rec.triggerHash;
+            address originalCommandAddress = rec.commandAddress;
```

Comment:
maybe this will be clearer
```suggestion
            TriggerRecord memory replaced = activeTriggers[replacedTriggerId];
            bytes32 replacedTriggersHash = replaced.triggerHash;
            address originalCommandAddress = replaced.commandAddress;
```